martin.yaml

```diff
@@ -7,15 +7,35 @@ max_feature_count: 500
 postgres:
     connection_string: ${DATABASE_URL}
     functions:
-        treatment_plan_prescriptions:
+        # layer with a series of stands, given a treatment_plan_id
+        # parameters:
+        # - treatment_plan_id: int
+        # - project_area_id: Optional[int]
+        dynamic_stands:
```

Comment:
Can find a better name for this?
treatment_stands? thats the required parameter so it kinda makes sense